components/KeysTableStatus.js

```diff
@@ -7,16 +7,17 @@ export const KeysTableStatus = ({ keys, account }) => {
     return (
         <Table variant='simple' size='sm' boxShadow="1px 1px 3px">
             <Tbody><Tr><Td>Weight</Td><Td>KeyId</Td><Td>Public Key</Td></Tr>
-                {keys.sort((a, b) => a.keyId > b.keyId ? 1 : -1).map(key => {
-                    return (<Tr backgroundColor={key?.sig ? 'lightGreen' : 'white'} key={`tr-${key.index}`} >
-                        <Td>{key.weight}</Td>
-                        <Td key={key.keyId}>{key.keyId}</Td>
-                        <Td>{abbrvKey(accountKeys[key.keyId])}</Td>
-                    </Tr>)
+                {keys.sort((a, b) => a.keyId > b.keyId ? 1 : -1).map((key, i) => {
```

Comment:
`.sort` will mutate en original `keys` array, are we ok with it?
if not, it's possible to make a shallow copy with `[...keys]`

Reply:
good suggestion, will update PR.